lib/zclient.h

```diff
@@ -441,7 +441,7 @@ struct zapi_nexthop {
 
 	struct ethaddr rmac;
 
-	uint32_t weight;
+	uint64_t weight;
```

Comment:
I'm a little concerned about this change, since zebra is only using 8 bits of the value sent via ZAPI?

Reply:
I will double check this and update you.

Reply:
to clarify -> The kernel only has 8 bits of weight.  So this would be `fine` since zebra already normalizes but you will need to normalize in zebra to an 8 bit number

Reply:
yes zebra normalizes it now.